src/EditorFeatures/Core/Implementation/BraceMatching/ClassificationTypeDefinitions.cs

```diff
@@ -6,6 +6,8 @@
 using Microsoft.VisualStudio.Text.Classification;
 using Microsoft.VisualStudio.Utilities;
 
+#pragma warning disable 649
```

Comment:
@jasonmalinowski looks like it needs to be in CoreCompileDependsOn instead of CompileDependsOn.  

Wondering if one day I'll find a CoreCompileDependsOnNoReallyItDoes